blog/_posts/2024-06-06-istio-native-sidecar.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Introducing Native Sidecar Support for Istio addon on AKS"
+description: "This blog talks about native sidecar support for Istio addon on AKS."
+date: 2024-06-06
+author: Fuyuan Bie
+categories: mesh
+---
+
+## Overview
```

Comment:
This heading is probably not necessary so removing this sub-heading to reduce spacing between page header and post content.

```suggestion
## Overview
```